### PR TITLE
Collect coredump

### PR DESCRIPTION
**Pull request by mimi1vx**
poo#51863

https://openqa.suse.de/tests/3331660

### CODE REVIEW

File: lib/opensusebasetest.pm [hunk 1]
```diff
@@ -927,6 +927,25 @@ sub select_user_serial_terminal {
...
+=head2 upload_coredumps
+
+ upload_coredumps();
+
+Upload all coredumps to logs
+=cut
+sub upload_coredumps {
+    my $res = script_run("coredumpctl -q", timeout => 10);
+    if (!$res) {
+        script_run("coredumpctl info | tee coredump-info.log");
```
Comment (plusky) [inline lib/opensusebasetest.pm]: Well thats very nice, but its far from usable coredump ...

File: lib/opensusebasetest.pm [hunk 2]
```diff
@@ -927,6 +927,25 @@ sub select_user_serial_terminal {
...
 }
 
+=head2 upload_coredumps
+
+ upload_coredumps();
+
+Upload all coredumps to logs
+=cut
+sub upload_coredumps {
+    my $res = script_run("coredumpctl -q", timeout => 10);
```
Comment (dzedro) [inline lib/opensusebasetest.pm]: Better run it with `--no-pager` ?

Comment (mimi1vx) [reply lib/opensusebasetest.pm]: ok, but still `  -q --quiet             Do not show info messages and privilege warning`

File: products/sle/main.pm [hunk 3]
```diff
@@ -886,6 +886,10 @@ elsif (get_var('LIBSOLV_INSTALLCHECK')) {
 elsif (get_var("EXTRATEST")) {
     boot_hdd_image;
     load_extra_tests();
+    loadtest "console/coredump_collect";
```
Comment (asmorodskyi) [inline products/sle/main.pm]: extratest infra is used by a lot of different projects currently , please next time when you plan to change flow like this for **everyone** inform more people. Quite unfortunate to find in wicked tests modules which I not confirm that they will be called :(
@Soulofdestiny , @rwx788 were you aware about this change ? is it ok for you ? 
at least I want to disabled for wicked , but let's clarify first what is more suitable - disable it for cases where maintainer is strongly against or just maybe enable only for cases it was expected to run. 
@baierjan what was you initial idea ? do flow did you have in mind ?

Comment (asmorodskyi) [reply products/sle/main.pm]: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/8410

Comment (mimi1vx) [reply products/sle/main.pm]: @asmorodskyi  why are you against coredump? This is standard debug information needed by developers, and the rest of is the correct shutdown of SUT

Comment (asmorodskyi) [reply products/sle/main.pm]: I am not "against coredumps" , I am against breaking test isolation . Each test need to test something certain , it is easier to maintain and understand the results. 
Regarding "correct" shutdown - can you explain me why I should care about graceful shutdown of something what will be wiped out in a second after that ?

**Pull request merged**